File: ElementExists.java
import java.util.Scanner;

public class ElementExists {

    public static boolean elementExists(int[] arr, int target) {
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] == target) {
                return true;
            }
        }
        return false;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        // Read number of test cases
        System.out.print("Enter number of test cases: ");
        int t = sc.nextInt();

        while (t-- > 0) {
            // Read size of array
            System.out.print("Enter size of array: ");
            int n = sc.nextInt();

            int[] arr = new int[n];

            // Read array elements
            System.out.println("Enter elements of array:");
            for (int i = 0; i < n; i++) {
                arr[i] = sc.nextInt();
            }

            // Read target element
            System.out.print("Enter target to search: ");
            int target = sc.nextInt();

            // Check if element exists
            if (elementExists(arr, target)) {
                System.out.println("Target found.");
            } else {
                System.out.println("Target not found.");
            }
        }

        sc.close();
    }
}

File: elementExists.js

function elementExists(arr, element) {
  // Loop through each element in the array
  for (let i = 0; i < arr.length; i++) {
    // Check if the current element is equal to the target element
    if (arr[i] === element) {
      return true; // If found, return true
    }
  }

  // If loop finishes without finding the element, return false
  return false;
}

// Example usage:
console.log(elementExists([1, 2, 3, 4], 3));  // true (3 exists)
console.log(elementExists([1, 2, 3, 4], 5));  // false (5 doesn't exist)
